core/src/main/kotlin/org/evomaster/core/search/FitnessValue.kt

```diff
@@ -153,13 +153,15 @@ class FitnessValue(
     }
 
     fun gqlErrors(idMapper: IdMapper, withLine : Boolean): List<String>{
-        return targets.keys
+        // GQLErrors would be 0.5 when it is initialed, so we count it when it is covered.
```

Comment:
avoid hardcoding values in comments (as those might change in the future). eg, instead of `be 0.5`, write something less brittle like `be >0`.

btw, what about the bug in the counting of queries/mutations?

Reply:
There was three bugs in calculating statistics:
In distinctActions (the counting of queries/mutations), gqlNoErrors and gqlErrors all of them are fixed now.

Reply:
was distinctActions  fixed in this branch? or in another one?

Reply:
It was fixed in another branch but I close it to avoid the repeated fix since when Man fix the gqlNoErrors and gqlErrors fix also distinctActions here.